Source/ProcessingTest/Presenters/ProcessingPresenter.cpp

```diff
@@ -168,7 +178,7 @@ void ProcessingPresenter::OpenRAWFile()
 {
     _view->EnableRendering(false);
 
-    QString fileName = QFileDialog::getOpenFileName(_view, tr("Open Image"), _lastDir, tr("DNG Files (*.dng *.DNG)"));
+    QString fileName = QFileDialog::getOpenFileName(_view, tr("Open Image"), _lastDir, tr("DNG Files (*.dng *.DNG *.mlv *.MLV)"));

```

Comment:
Why is MLV format under DNG Files? It would be better to add additional filter and afterwards add another filter something like "Supported formats":

QFileDialog dialog(_view, tr("Open Image"), _lastDir, tr("DNG Files (*.dng *.DNG);;MagicLantern Files (*.mlv *.MLV)"));

Reply:
Corrected